src/mod/applications/mod_conference/mod_conference.c

```diff
@@ -1343,6 +1345,38 @@ void conference_xlist(conference_obj_t *conference, switch_xml_t x_conference, i
 
 		switch_snprintf(tmp, sizeof(tmp), "%d", member->volume_out_level);
 		add_x_tag(x_member, "output-volume", tmp, toff++);
+
+		// relationships
+		x_relationships = switch_xml_add_child_d(x_member, "relationships", toff++);
+		switch_assert(x_relationships);
+
+		for (rel = member->relationships, roff = 0; rel; rel = rel->next, roff++) {
+			uint32_t count = 0;
+			char id[30] = "";
+
+			if (rel->id == member->id) {
+				switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_CRIT, "Same target.\n");
+				continue;
+			}
+
+			x_relationship = switch_xml_add_child_d(x_relationships, "relationship", roff);
+			switch_assert(x_relationship);
+
+			switch_snprintf(id, sizeof(id), "%d", member->id);
```

Comment:
This should use rel->id rather than member->id. Currently all relationships of a given member will be reported with same id (of that member) but id of the other member (to whom relationship pertains) should be printed. 

Change to `switch_snprintf(id, sizeof(id), "%d", rel->id);` please.

Reply:
Other than this looks good to me.